lib/acts_as_scrubbable.rb

```diff
@@ -45,16 +52,18 @@ def self.scrub_map
       :state_abbr        => -> { Faker::Address.state_abbr },
       :state             => -> { Faker::Address.state },
       :city              => -> { Faker::Address.city },
+      :full_address      => -> { Faker::Address.full_address },
       :latitude          => -> { Faker::Address.latitude },
       :longitude         => -> { Faker::Address.longitude },
       :username          => -> { Faker::Internet.user_name },
       :boolean           => -> { [true, false ].sample },
-      :school            => -> { Faker::University.name }
+      :school            => -> { Faker::University.name },
+      :bs                => -> { Faker::Company.bs },
+      :phone_number      => -> { Faker::PhoneNumber.phone_number }
```

Comment:
i personally found using these built-ins just some of the time seemed confusing, so didn't use them at all.

Reply:
after looking at the other PR i see you used `:bs` a bunch, so makes sense to put it either here or in the initializer config in that case.